processing/src/main/java/io/druid/query/search/search/DefaultSearchQueryMetrics.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.query.search.search;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.druid.query.DefaultQueryMetrics;
+
+public class DefaultSearchQueryMetrics extends DefaultQueryMetrics<SearchQuery> implements SearchQueryMetrics
```

Comment:
Please follow the 6-step procedure defined in the "Making subinterfaces of QueryMetrics for emitting custom dimensions and/or metrics for specific query types" section in `QueryMetrics` docs. Same for `SelectQueryMetrics`. Also, please update that section to use another query type as an example, because "Search" won't be a query time without specific QueryMetrics any more, as assumed in that doc.